com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineAsset.Migration.cs

```diff
@@ -204,10 +205,19 @@ enum Version
                 data.m_RenderPipelineSettings.m_ObsoleteDecalLayerName6 = null;
                 data.m_RenderPipelineSettings.m_ObsoleteDecalLayerName7 = null;
 #pragma warning restore 618
-            }) ,
+            }),
             MigrationStep.New(Version.DecalSurfaceGradient, (HDRenderPipelineAsset data) =>
             {
                 data.m_RenderPipelineSettings.supportSurfaceGradient = false;
+            }),
+            MigrationStep.New(Version.MovedSupportRuntimeDebugDisplayToGlobalSettings, (HDRenderPipelineAsset data) =>
+            {
+#pragma warning disable 618 // Type or member is obsolete
+                HDRenderPipelineGlobalSettings.instance.supportRuntimeDebugDisplay = data.m_RenderPipelineSettings.m_ObsoleteSupportRuntimeDebugDisplay;
+#if UNITY_EDITOR
+                UnityEditor.EditorUtility.SetDirty(HDRenderPipelineGlobalSettings.instance);
+#endif
```

Comment:
I'm not sure if this is a good thing to do. I must set the global asset dirty to ensure to ensure it gets saved. Or is this code in the wrong place and I should instead increment the version on the GlobalSettins asset?

Reply:
I'm now realizing there could be multiple RP assets, and whatever gets migrated last would remain in effect on the global asset. So I guess I'm doing this backward now - I should instead migrate the global asset to a new version, and just look up the currently active RP asset for the value.